test/integration/testdata/check.proxy.golden

```diff
@@ -63,6 +63,10 @@ linkerd-control-plane-proxy
 √ control plane proxies are up-to-date
 √ control plane proxies and cli versions match
 
+linkerd-opaque-ports-definition
+-------------------------------
+√ the opaque-ports labels have been properly defined
```

Comment:
hmmm wonder if this is why you were seeing the failures; labels have been changed for annotations so the test suite might not like it 🤔

Reply:
I am not really sure, looking at the output data in the CI

Expected:

```
Linkerd core checks
        ===================
        
        kubernetes-api
        --------------
        √ can initialize the client
        √ can query the Kubernetes API
        
        kubernetes-version
        ------------------
        √ is running the minimum Kubernetes API version
        √ is running the minimum kubectl version
        
        linkerd-existence
        -----------------
        √ 'linkerd-config' config map exists
        √ heartbeat ServiceAccount exist
        √ control plane replica sets are ready
        √ no unschedulable pods
        √ control plane pods are ready
        
        linkerd-config
        --------------
        √ control plane Namespace exists
        √ control plane ClusterRoles exist
        √ control plane ClusterRoleBindings exist
        √ control plane ServiceAccounts exist
        √ control plane CustomResourceDefinitions exist
        √ control plane MutatingWebhookConfigurations exist
        √ control plane ValidatingWebhookConfigurations exist
        √ control plane PodSecurityPolicies exist
        
        linkerd-identity
        ----------------
        √ certificate config is valid
        √ trust anchors are using supported crypto algorithm
        √ trust anchors are within their validity period
        √ trust anchors are valid for at least 60 days
        √ issuer cert is using supported crypto algorithm
        √ issuer cert is within its validity period
        √ issuer cert is valid for at least 60 days
        √ issuer cert is issued by the trust anchor
        
        linkerd-webhooks-and-apisvc-tls
        -------------------------------
        √ proxy-injector webhook has valid cert
        √ proxy-injector cert is valid for at least 60 days
        √ sp-validator webhook has valid cert
        √ sp-validator cert is valid for at least 60 days
        
        linkerd-version
        ---------------
        √ can determine the latest version
        √ cli is up-to-date
        
        control-plane-version
        ---------------------
        √ can retrieve the control plane version
        √ control plane is up-to-date
        √ control plane and cli versions match
        
        linkerd-control-plane-proxy
        ---------------------------
        √ control plane proxies are healthy
        √ control plane proxies are up-to-date
        √ control plane proxies and cli versions match
        
        linkerd-opaque-ports-definition
        -------------------------------
        √ the opaque-ports labels have been properly defined
        
        Status check results are √
```

Actual:

```
Linkerd core checks
        ===================
        
        kubernetes-api
        --------------
        √ can initialize the client
        √ can query the Kubernetes API
        
        kubernetes-version
        ------------------
        √ is running the minimum Kubernetes API version
        √ is running the minimum kubectl version
        
        linkerd-existence
        -----------------
        √ 'linkerd-config' config map exists
        √ heartbeat ServiceAccount exist
        √ control plane replica sets are ready
        √ no unschedulable pods
        √ control plane pods are ready
        
        linkerd-config
        --------------
        √ control plane Namespace exists
        √ control plane ClusterRoles exist
        √ control plane ClusterRoleBindings exist
        √ control plane ServiceAccounts exist
        √ control plane CustomResourceDefinitions exist
        √ control plane MutatingWebhookConfigurations exist
        √ control plane ValidatingWebhookConfigurations exist
        √ control plane PodSecurityPolicies exist
        
        linkerd-identity
        ----------------
        √ certificate config is valid
        √ trust anchors are using supported crypto algorithm
        √ trust anchors are within their validity period
        √ trust anchors are valid for at least 60 days
        √ issuer cert is using supported crypto algorithm
        √ issuer cert is within its validity period
        √ issuer cert is valid for at least 60 days
        √ issuer cert is issued by the trust anchor
        
        linkerd-webhooks-and-apisvc-tls
        -------------------------------
        √ proxy-injector webhook has valid cert
        √ proxy-injector cert is valid for at least 60 days
        √ sp-validator webhook has valid cert
        √ sp-validator cert is valid for at least 60 days
        
        linkerd-version
        ---------------
        √ can determine the latest version
        √ cli is up-to-date
        
        control-plane-version
        ---------------------
        √ can retrieve the control plane version
        √ control plane is up-to-date
        √ control plane and cli versions match
        
        linkerd-control-plane-proxy
        ---------------------------
        √ control plane proxies are healthy
        √ control plane proxies are up-to-date
        √ control plane proxies and cli versions match
        
        linkerd-opaque-ports-definition
        -------------------------------
        √ the opaque-ports annotations have been properly defined
        
        Status check results are √
        
        Linkerd extensions checks
        =========================
        
        linkerd-viz
        -----------
        √ linkerd-viz Namespace exists
        √ linkerd-viz ClusterRoles exist
        √ linkerd-viz ClusterRoleBindings exist
        √ tap API server has valid cert
        √ tap API server cert is valid for at least 60 days
        √ tap API service is running
        √ linkerd-viz pods are injected
        √ viz extension pods are running
        √ viz extension proxies are healthy
        ‼ viz extension proxies are up-to-date
            some proxies are not running the current version:
        	* grafana-57487b6558-64bb5 (git-e5a319e5)
        	* tap-injector-59f7489dd9-ssvcm (git-e5a319e5)
        	* web-6885bcbb8f-fxj5j (git-e5a319e5)
        	* tap-674f668557-72hkj (git-e5a319e5)
        	* metrics-api-5ccb85df68-9v8p4 (git-e5a319e5)
            see https://linkerd.io/2/checks/#l5d-viz-proxy-cp-version for hints
        √ viz extension proxies and cli versions match
        √ can initialize the client
        √ viz extension self-check
        
        Status check results are √
```

Not sure where the `linkerd-viz` diff comes from ...